scripts/release/rpm/pipeline_azurelinux.sh

```diff
@@ -11,11 +11,15 @@ set -exv
 
 CLI_VERSION=`cat src/azure-cli/azure/cli/__main__.py | grep __version__ | sed s/' '//g | sed s/'__version__='// |  sed s/\"//g`
 
+# PIP_INDEX_URL env must exist in `docker build --secret`, use an empty string if it doesn't exist.
+export PIP_INDEX_URL=${PIP_INDEX_URL}
```

Comment:
Error message without empty env: `ERROR: failed to stat PIP_INDEX_URL: stat PIP_INDEX_URL: no such file or directory`